.cookiecutter.json

```diff
@@ -0,0 +1,25 @@
+{
+  "cookiecutter": {
+    "codeowner_github_usernames": "@glennmatthews @jathanism @lampwins @chadell",
+    "full_name": "Network to Code, LLC",
+    "email": "info@networktocode.com",
+    "github_org": "nautobot",
+    "plugin_name": "nautobot_ssot",
+    "verbose_name": "Nautobot Ssot",
+    "plugin_slug": "nautobot-ssot",
+    "project_slug": "nautobot-plugin-ssot",
+    "repo_url": "https://github.com/nautobot/nautobot-plugin-ssot",
+    "base_url": "ssot",
+    "min_nautobot_version": "1.4.0",
+    "max_nautobot_version": "1.9999",
+    "nautobot_version": "1.5.1",
+    "camel_name": "NautobotSsot",
+    "project_short_description": "Nautobot Single Source of Truth",
+    "version": "1.2.0",
```

Comment:
Should this be 1.3.0? I'm not entirely clear on the purpose of this file - do we need to keep this field in sync with the app version number?

Reply:
This file is used to re-generate the cookie I believe. I'm not entirely sure it's required to be updated as we update the repo. Thoughts @Kircheneer and @chadell?

Reply:
Paging @snaselj